src/main/java/edu/ie3/datamodel/io/factory/EntityData.java

```diff
@@ -80,7 +80,7 @@ public String getField(String field) {
     if (!fieldsToAttributes.containsKey(field))
       throw new FactoryException(String.format("Field \"%s\" not found in EntityData", field));
 
-    return fieldsToAttributes.get(field);
+    return fieldsToAttributes.get(field).replaceAll("\"\"", "\"");
```

Comment:
I think this is not comprehensive, as e.g. the surrounding quotes are not treated (see comment in `BufferedCsvWriter`).